README.md

```diff
@@ -29,30 +29,32 @@ The plugin is used in the [snap framework] (http://github.com/intelsdi-x/snap).
 
 ### Operating systems
 
-All OSs currently supported by snap:
+All OSs currently supported by Snap:
 * Linux/amd64
 
 ### Installation
+#### Download the plugin binary:
+
+You can get the pre-built binaries for your OS and architecture from the plugin's [GitHub Releases](https://github.com/intelsdi-x/snap-plugin-collector-influxdb/releasess) page. Download the plugin from the latest release and load it into `snapd` (`/opt/snap/plugins` is the default location for Snap packages).
 
 #### To build the plugin binary:
 
 Fork https://github.com/intelsdi-x/snap-plugin-collector-influxdb
-
 Clone repo into `$GOPATH/src/github.com/intelsdi-x/`:
 
 ```
 $ git clone https://github.com/<yourGithubID>/snap-plugin-collector-influxdb.git
 ```
 
-Build the snap influxdb collector plugin by running make within the cloned repo:
+Build the Snap influxdb plugin by running make within the cloned repo:
 ```
 $ make
 ```
-This builds the plugin in `/build/rootfs/`
+This builds the plugin in `./build/`
 
 ### Configuration and Usage
 
-* Set up the [snap framework](https://github.com/intelsdi-x/snap/blob/master/README.md#getting-started)
+* Set up the [Snap framework](https://github.com/intelsdi-x/snap/blob/master/README.md#getting-started)
 * Load the plugin and create a task, see example in [Examples](https://github.com/intelsdi-x/snap-plugin-collector-influxdb/blob/master/README.md#examples).
```

Comment:
removing `blob/master` will lead to 404 Not Found. In `Snap` documentation this pattern for url is used as well